index.html

```diff
@@ -5,8 +5,43 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
+    <link href="/styles/index.css" rel="stylesheet" />
 </head>
 <body>
+    <header>
+        <p id="sky-emoji">☁️ ☁️ ☁️ ☀️ ☁️ ☁️ ☀️ ☁️ ☁️ ☀️ ☁️ ☁️ ☁️</p>
+        <h1 id="weather-report">Weather Report</h1>
+        
+    </header>
+    <main >
+        <ul class="box-container">
+            <li id="temp-title">Temperature</li>
```

Comment:
This would make sense as a `h2` since it contains some title information:

```suggestion
            <h2 id="temp-title">Temperature</h2>
```